core/src/forest/ForestTrainers.h

```diff
@@ -29,6 +29,12 @@ ForestTrainer quantile_trainer(const std::vector<double>& quantiles);
 
 ForestTrainer regression_trainer();
 
+ForestTrainer local_linear_trainer(double split_lambda,
```

Comment:
Maybe we should name this `ll_regression_trainer` to be consistent with the methods in `ForestPredictors` like `ll_regression_predictor`.